### PR TITLE
Complete refactor

### PR DESCRIPTION
**Pull request by gibsramen**

### CODE REVIEW

File: evident/effect_size.py [hunk 1]
```diff
@@ -1,157 +1,129 @@
...
+        values_dict[grp] = dh.subset_values(_df.index)
+
+    for grp1, grp2 in combinations(values_dict.keys(), 2):
+        vals1 = values_dict[grp1]
+        vals2 = values_dict[grp2]
+        effect_size = calculate_cohens_d(vals1, vals2)
+        res = PairwiseEffectSizeResult(effect_size, col, grp1, grp2)
+        col_results.append(res)
+    return col_results
+>>>>>>> new-repo/main
```
Comment (antgonza) [inline evident/effect_size.py]: This shouldn't be here

File: evident/tests/test_diversity_handler.py [hunk 2]
```diff
@@ -0,0 +1,237 @@
+import os
```
Comment (antgonza) [inline evident/tests/test_diversity_handler.py:1]: Fine to leave it like this but out of curiosity, why not use a different test_*_.py file for each object?

File: setup.py [hunk 3]
```diff
@@ -1,57 +1,62 @@
...
-    hit = _version_re.search(f.read().decode('utf-8')).group(1)
-    __version__ = str(ast.literal_eval(hit))
+with open("README.md") as f:
+    long_description = f.read()
 
+q2_cmds = ["q2-evident=evident.q2.plugin_setup:plugin"]
 
 classes = """
-    Development Status :: 4 - Beta
+    Development Status :: 3 - Alpha
```
Comment (antgonza) [inline setup.py]: Should this be Beta?

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9"]
```
Comment (antgonza) [inline .github/workflows/main.yml]: nice.

**Review by antgonza — CHANGES_REQUESTED** [review]
This is great, thank you! A few comments, could you also add flake8 to the code?

**Comment by antgonza** [comment]
BTW I pushed main.yml to master so you pull from to your branch and (AFAIK) that should trigger an action.

File: evident/tests/test_diversity_handler.py (hunk 2 above)
Comment (gibsramen) [reply evident/tests/test_diversity_handler.py:1]: Probably would have been better to do that.

File: setup.py (hunk 3 above)
Comment (gibsramen) [reply setup.py]: Yup

**Review by antgonza — APPROVED** [review]

**Pull request merged**